nixos/modules/virtualisation/hyperv-guest.nix

```diff
@@ -22,6 +22,18 @@ in {
           Server 2008 R2 or earlier.
         '';
       };
+
+      enchanchedSessionMode = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable enchanched session mode.";
+      };
+
+      x11 = mkOption {
```

Comment:
mkEnableOption

Reply:
Cannot use `mkEnableOption`, since the option is disabled by default.

Reply:
see above.